src/docs/PCW/PCWUserGuide.adoc

```diff
@@ -254,7 +254,7 @@ The possible actions are:
 
 Both examples are provided below.
 
-==== Submit to ProActive Scheduler as Rule’s Action
+==== Submit to ProActive Scheduler as rule’s action
```

Comment:
==== Submit to ProActive Scheduler as rule action <- American english, no " 's "

Reply:
I don't understand @mcompastie's comment. I think both American and British English use _'s_ for possessives, don't they?
Actually, which variation are we meant to use for documentation? (honest question)

Reply:
@luispinedamx in this PR actually I try to use all over the document the lowercase "r" for "rule". The idea is to have consistente lowercase in the docmentation.
Where did you see with uppercase?

Reply:
@mcompastie do you believe that "Submit to ProActive Scheduler as rule action" is better?
what is the problem with " 's " ?

Reply:
@ShatalovYaroslav Please see https://en.wikipedia.org/wiki/English_possessive#Nouns_and_noun_phrases : `In the case of singular nouns ending in -s, the possessive was traditionally also spelled by adding only an apostrophe (despite often being pronounced differently), but this is now generally discouraged in American and Canadian English:[3]`

Reply:
@luispinedamx Since there is the word "visualization" and not "visualisation", "behavior" and not "behaviour", and no " 's ", I think this document is redacted in american English.

Reply:
@mcompastie thanks for information. But I believe that we are targeting British English, not American one.

Reply:
@mcompastie I am genuinely curious about the _'s_ thing, I am even reading the source from that wikipedia entry that you cite ([Purdue OWL](https://owl.purdue.edu/owl/general_writing/punctuation/apostrophe_introduction.html), Purdue, an American University). In most cases, the possessive is done by adding _'s_, except when the noun is plural and ends in _s_, in such a case we just use the apostrophe. I am open to discussion ;) , but probably not in this thread.